src/style.css

```diff
@@ -8,11 +8,16 @@ body {
   justify-content: center;
 }
 
+.stars:last-child {
+  margin-right: 0;
+}
```

Comment:
Stats this is your Block but you have margin-right inside element of this block so for remove it from last element you should use this pseudo class for element and not for block